### PR TITLE
[sonos] Add a rule example to start playing a favorite

### PR DESCRIPTION
**Pull request by joke24**
added example in documentation on how to play favorites.

### CODE REVIEW

File: bundles/org.openhab.binding.sonos/README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ The devices support the following channels:
...
 | currenttrackuri     | String    | R           | URI of the current track                                                                                                                                  | all                                  |
 | currenttransporturi | String    | R           | URI of the current AV transport                                                                                                                           | all                                  |
-| favorite            | String    | W           | Play the given favorite entry. The favorite entry has to be predefined in the Sonos Controller app                                                        | all                                  |
+| favorite            | String    | W           | Play the given favorite entry. The favorite entry has to be predefined in the Sonos Controller app. Use the exact name of the favorite shown in the app for start playing from rules.                                                      | all                                  |
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sonos/README.md:72]: Probably rather "... shown in the app **to** start playing..."
By "app", you mean the Sonos app ?

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.sonos/README.md [hunk 2]
```diff
@@ -166,3 +167,14 @@ sitemap demo label="Main Menu"
 		}
 }
 ```
+
+sonos.rules:
```
Comment (jlaur) [inline bundles/org.openhab.binding.sonos/README.md:171]: For consistency, either rename this to demo.rules, or rename the others to sonos.things, sonos.items and sonos.sitemap, respectively.

File: bundles/org.openhab.binding.sonos/README.md (hunk 1 above)
Comment (joke24) [reply bundles/org.openhab.binding.sonos/README.md:72]: yes, thanks for the feedback. this is my first addition to the documentation. I'm learning every day :)

File: bundles/org.openhab.binding.sonos/README.md (hunk 2 above)
Comment (joke24) [reply bundles/org.openhab.binding.sonos/README.md:171]: yes, thanks for the feedback. this is my first addition to the documentation. I'm learning every day :)

**Pull request closed without merging**